AppCenterReactNativeShared/Products/local.podspec

```diff
@@ -14,5 +14,5 @@ Pod::Spec.new do |s|
   s.platform          = :ios, '9.0'
   s.requires_arc      = true
   s.vendored_frameworks = 'AppCenterReactNativeShared/AppCenterReactNativeShared.framework'
-  s.dependency 'AppCenter/Core', '2.2.1-5'
+  s.dependency 'AppCenter/Core', '2.2.0'
```

Comment:
we need the prerelease, please leave that version as is.

Reply:
Changed